cmd/crc/cmd/setup.go

```diff
@@ -58,7 +58,7 @@ func runSetup(arguments []string) error {
 	}
 
 	if !config.Get(crcConfig.Bundle).IsDefault {
-		if err := validation.ValidatePath(config.Get(crcConfig.Bundle).AsString()); err != nil {
+		if err := validation.ValidateBundlePath(config.Get(crcConfig.Bundle).AsString(), crcConfig.GetPreset(config)); err != nil {
```

Comment:
this is already part of https://github.com/code-ready/crc/pull/3315/commits/030b0275998fb7f545aa2476095790d9bcc51d28 and will be removed one other merge or I will cherry pick it.